CMakeLists.txt

```diff
@@ -340,10 +340,10 @@ endif()
 option(BUILD_PYTHON_SUPPORT "Build Python gadgets etc" On)
 
 if (BUILD_PYTHON_SUPPORT)
-    find_package(Python3 COMPONENTS Development NumPy REQUIRED)
+    find_package(Python3 3.9.7 EXACT COMPONENTS Interpreter Development NumPy REQUIRED)
```

Comment:
Doh - missed that!

So there are a couple of ways around this - (i) bracket that statement with a cmake appropriate "ifdef macOS" to run that (since this **only** works in Conda on the mac, for the time being), and have the "else" portion of the "ifdef" be as before; or (ii) temporarily disable Python support in the macOS build, where this bit of cmake is not executed.  The latter is probably my preferred path.